apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -246,6 +253,10 @@ def initialize(self):
         self._l3_plugin = None
         self._db_plugin = None
         self.attestator = attestation.EndpointAttestator(self.apic_manager)
+        net_cons_source = cfg.CONF.ml2_cisco_apic.network_constraints_filename
```

Comment:
Why having a different file instead of using one of the existing config files? Is this file using a different syntax than the typical openstak .ini file? Each deployment can choose to put net-conf in a separated file and provide the path to the Neutron server run script if they want to.